### ISSUE TITLE
Config window disappeared after update

### ISSUE THREAD

**Issue by ssspacez** [psifidotos/Latte-Dock#972]
`0.7.79+git20180407.1618-0~ubuntu18.04~ppa1`

![window](https://user-images.githubusercontent.com/34775187/38473140-2bc84f72-3b61-11e8-9c3d-6c3734b8663b.png)

**Comment by psifidotos** [comment]
alternative you can run 'latte-dock -d' some warning or errors should appear when the config window isnt showing.

**Comment by ssspacez** [comment]
How do I clean the dock cache? I tried ending the process with system monitor, closed the session and came back but the issue is not solved.

**Comment by psifidotos** [comment]
1. close Latte
2. remove everything at `~/.cache/lattedock`
3. start Latte

**Comment by ssspacez** [comment]
Didn't work either.

```
latte-dock -d
debug 19:40:15.178178] - KActivities: Database connection:  "kactivities_db_resources_140714800625664_readonly" 
    query_only:          QVariant(qlonglong, 1) 
    journal_mode:        QVariant(QString, "wal") 
    wal_autocheckpoint:  QVariant(qlonglong, 100) 
    synchronous:         QVariant(qlonglong, 0)
[debug 19:40:15.179179] - Closing SQL connection:  "kactivities_db_resources_140714800625664_readonly"
[warning 19:40:19.779779] - QQmlComponent: Component is not ready
[debug 19:40:19.7878] - dock config view : initialization started...
[warning 19:40:19.838838] - file:///usr/share/plasma/shells/org.kde.latte.shell/contents/configuration/LatteDockConfiguration.qml:379:21: Type AppearanceConfig unavailable 
                         AppearanceConfig { 
                         ^
[warning 19:40:19.838838] - file:///usr/share/plasma/shells/org.kde.latte.shell/contents/configuration/AppearanceConfig.qml:161:17: Type LatteExtraControls.Slider unavailable 
                     LatteExtraControls.Slider { 
                     ^
[warning 19:40:19.838838] - file:///usr/share/plasma/shells/org.kde.latte.shell/contents/controls/Slider.qml:24:1: module "QtQuick.Controls" version 1.6 is not installed 
     import QtQuick.Controls 1.6 as QtControls 
     ^
[debug 19:40:19.839839] - dock config view : initialization ended...
[debug 19:40:38.0909] - DockConfigView deleting ...
```

**Comment by psifidotos** [comment]
What is you Qt version?

**Comment by psifidotos** [comment]
Is package qtcontrols installed?

**Comment by ssspacez** [comment]
Desktop: KDE Plasma 5.12.4 (Qt 5.9.4) 
Display Server: x11 (X.Org 1.19.6 )

**Comment by psifidotos** [comment]
qml-module-qtquick-controls is called in Neon. Is that installed in your system?

**Comment by ssspacez** [comment]
It's Kubuntu 18.04 Beta 2, apparently yes they are installed:

```
qml-module-qtquick-controls/bionic,now 5.9.4-0ubuntu1 amd64 [installed]
  Qt 5 Quick Controls QML module

qml-module-qtquick-controls-styles-breeze/bionic,now 4:5.12.4-0ubuntu1 amd64 [installed]
  QtQuick style for Qt and KDE Software

qml-module-qtquick-controls2/bionic,now 5.9.4-0ubuntu1 amd64 [installed]
  Qt 5 Qt Quick Controls 2 QML module

```

**Comment by ssspacez** [comment]
Using Wayland is almost the same behavior, but the blurred window on the top left is bigger, a vertical rectangle covering more than half of the left side.

Can't post a screenshot here because I'm not being able to take a screenshot with a timer with Spectacle on Wayland, maybe it's a bug(it's Kubuntu beta 2).

**Comment by psifidotos** [comment]
not an easy replacement the Sliders... I reverted the change I must confirm if this is a version issue or ubuntu packaging issue.

**Comment by ssspacez** [comment]
Fixed, 0.7.79+git20180409.2337-0 it's working again.
